src/UserInfo/UserInfo.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import request from '../api-requests.js';
+
+class UserInfo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoggedIn: this.props.isLoggedIn || false,
+      isRejected: false,
+      userID: '',
+      name: '',
+      password: '',
+      idNumber: null
+    }
+  }
+
+  render() {
```

Comment:
Thank you for running me through the login functionality via a call!! I really appreciated it, and it makes a lot of sense for me now. Looks great!